src/main/cpp/RobotContainer.cpp

```diff
@@ -51,9 +52,12 @@ RobotContainer::RobotContainer() {
                 m_driverController.GetLeftX(), OIConstants::kDriveDeadband)},
             -units::radians_per_second_t{frc::ApplyDeadband(
                 m_driverController.GetRightX(), OIConstants::kDriveDeadband)},
-            true, true, kLoopTime);
+            true, false, kLoopTime);
       },
       {&m_drive}));
+
+    m_chooser.SetDefaultOption("Leave Community", pathplanner::PathPlannerAuto("Leave COM").ToPtr().Unwrap().get());
```

Comment:
Extract to constants?

Reply:
Extract what?

Reply:
The names `"Leave Community"` and `"Leave COM"`

Reply:
Those are arguments?